app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -58,7 +58,7 @@ RDEPEND="!app-emulation/rocket
 
 BUILDDIR="build-${P}"
 
-STAGE1INSTALLDIR="/usr/share/rkt/"
+STAGE1INSTALLDIR="/usr/$(get_libdir)/rkt/stage1-images"
```

Comment:
I vote for changing to the rkt default, compatibility can be handled by a symlink.